glide.yaml

```diff
@@ -12,7 +12,7 @@ import:
 - package: github.com/tyler-smith/go-bip39
 - package: github.com/FactomProject/basen
 - package: github.com/ontio/ontology
-  version: go-sdk-v1.7.0
+  version: v1.8.1
```

Comment:
why not latest version?

Reply:
sdk should use the version running on mainnet